src/javax/microedition/media/Manager.java

```diff
@@ -26,8 +27,9 @@
 	public static final String TONE_DEVICE_LOCATOR = "device://tone";
 
 
-	public static Player createPlayer(InputStream stream, String type) throws MediaException
+	public static Player createPlayer(InputStream stream, String type) throws IOException, MediaException
 	{
+		if(stream == null) { throw new IllegalArgumentException(); } 
```

Comment:
This check isn't necessary as you already check for null before you call this in `javax.microedition.media.Manager.createPlayer`.

Reply:
Very well, i'll be removing it.